src/api_routes_v4.rs

```diff
@@ -387,6 +404,17 @@ pub fn config(cfg: &mut ServiceConfig, rate_limit: &RateLimitCell) {
           .wrap(rate_limit.register())
           .route("/authenticate", post().to(authenticate_with_oauth)),
       )
-      .route("/sitemap.xml", get().to(get_sitemap)),
+      .service(
+        scope("/image")
+          .service(
+            resource("")
+              .wrap(rate_limit.image())
+              .route(post().to(upload_image))
+              .route(delete().to(delete_image)),
+          )
+          .route("/proxy", get().to(image_proxy))
+          .route("/{filename}", get().to(get_image))
```

Comment:
I guess the filename one is the same, but the `/proxy` is new, correct? That means we're going to need to do a DB migration to correct all the past ones, or we're going to have broken proxied images.